airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -22,11 +22,13 @@
 from pathlib import Path
```

Comment:
Thanks for the reviews @shahar1. The changes on core and providers are very related, and separate PRs won't make much sense. Also, this change won't appear in the changelog since it's part of an AIP and would be skipped.